### PR TITLE
Added displays for death count and last IL time

### PR DESCRIPTION
**Pull request by Thermospore**

### CODE REVIEW

**Review by negative-seven — DISMISSED** [review]
[removed]

File: LiveSplit.SuperMeatBoy.asl [hunk 1]
```diff
@@ -87,11 +92,61 @@ init
...
+		// Truncate
+		string t = "";
+		for (int i = 0; i < finalLength; i++)
+		{
+			t += s[i];
+		}
+		s = t;
+		
+		// Display final string
+		vars.SetTextComponent(name, s, settings["createUI"]);
```
Comment (negative-seven) [inline LiveSplit.SuperMeatBoy.asl]: You can use `String.Format("{0:0.000}", f)`

**Review by negative-seven — CHANGES_REQUESTED** [review]

File: LiveSplit.SuperMeatBoy.asl [hunk 2]
```diff
@@ -29,6 +30,9 @@ startup
 		string description = String.Format("Split before boss {0}", world);
 		settings.Add(name, false, description, "bossSplit");
 	}
+		
+	settings.Add("deathDisp", false, "Death count display");
+	settings.Add("createUI", true, "Create UI if needed");
```
Comment (negative-seven) [inline LiveSplit.SuperMeatBoy.asl]: Could there perhaps be a way to omit needing createUI? Seems not too user-friendly the way it is now.

File: LiveSplit.SuperMeatBoy.asl [hunk 3]
```diff
@@ -23,20 +23,16 @@ startup
...
+			string name = String.Format("boss{0}Split", world);
+			string description = String.Format("Split before boss {0}", world);
+			settings.Add(name, false, description, "bossSplit");
+		}
 	settings.Add("deathDisp", false, "Death count display");
-	
 	settings.Add("ilDisp", false, "Last IL Time display");
-	settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
-	
+		settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
```
Comment (negative-seven) [inline LiveSplit.SuperMeatBoy.asl:35]: I'm a bit confused by these formatting changes?

Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl:35]: I arranged the settings in a hierarchy, so any settings/code that are a subset of their parent setting are indented

File: LiveSplit.SuperMeatBoy.asl (hunk 1 above)
Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl]: Thanks! Yea, I suspected there was a more efficient way to do that...

File: LiveSplit.SuperMeatBoy.asl (hunk 2 above)
Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl]: Yea this setting is a holdover from a previous implementation, I suppose it isn't neccessary. I can just remove it

File: LiveSplit.SuperMeatBoy.asl (hunk 3 above)
Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl:35]: I put the address list in ascending order with the addresses all lined up, which is a format I've found to be very readable and easy to work with

File: LiveSplit.SuperMeatBoy.asl (hunk 1 above)
Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl]: Fixed!

File: LiveSplit.SuperMeatBoy.asl (hunk 2 above)
Comment (Thermospore) [reply LiveSplit.SuperMeatBoy.asl]: Fixed!

File: LiveSplit.SuperMeatBoy.asl (hunk 3 above)
Comment (negative-seven) [reply LiveSplit.SuperMeatBoy.asl:35]: The address list is a nice change, but additional indenting I think can be a bit misleading, making it look like a code block. I'll merge this right now, though.

**Comment by negative-seven** [comment]
Thanks for contributing!

**Pull request merged**